.gitignore

```diff
@@ -10,3 +10,4 @@ venv
 .env
 bigtable-admin.json
 bigtable-writer.json
+**/cert.pem
```

Comment:
Where do you obtain this file from?

Reply:
The system ca root store. I can show you

Reply:
Enterprise companies may have an internal certificate authority (which often results in SSL errors). If you are inside one such enterprise, you can get this from your organization.